### PR TITLE
Add Fortiq Feature Documentation

### PR DESCRIPTION
**Pull request by vertiq-jordan**
Added GPIO, ADC, PWM output, CANOpen, and step/dir documentation into the feature manual. Had to add a new css file to format the GPIO register table how I wanted. The read-the-docs page is hidden from public view, but you should be able to get it from here: https://iqmotion.readthedocs.io/en/add_fortiq_docs/

### CODE REVIEW

File: docs/_static/tight_table.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (vertiq-jordan) [inline docs/_static/tight_table.css:1]: a new css file that takes a large table and forces it to fit on the page without the need for a horizontal scroll bar.

File: docs/conf.py [hunk 2]
```diff
@@ -70,4 +70,5 @@
 
 html_logo = "_static/VertiqClear.png"
 
-html_css_files = ['custom_width.css', 'bottom_caption.css']
\ No newline at end of file
+html_css_files = ['custom_width.css', 'bottom_caption.css', 'tight_table.css']
```
Comment (vertiq-jordan) [inline docs/conf.py:73]: adds the new css to the build process so we can use it

File: docs/index.rst
Comment (vertiq-jordan) [inline docs/index.rst:1]: added all of the new features to the document

File: docs/manual/manual_adc_interface.rst
Comment (vertiq-jordan) [inline docs/manual/manual_adc_interface.rst:1]: ADC Interface manual. Tells people how to use it both through the control center and the python api

File: docs/manual/manual_gpio_interface.rst [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+	For example, to save the value in the *modes_register*
+
+	.. code-block::
+		
+		fortiq.save("gpio_controller", "modes_register")
+
+.. _gpio interface client:
+
+GPIO Controller Client - Entry Summary
```
[the remark this answers is not in the document]
Comment (vertiq-jordan) [reply docs/manual/manual_gpio_interface.rst:189]: Decision for the ADC on will decide what to do with this

File: docs/manual/manual_adc_interface.rst [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+		
+		print(fortiq.get("adc_interface", "adc_voltage"))
+
+#. To read the raw value, use: 
+
+	.. code-block::
+
+		print(fortiq.get("adc_interface", "raw_value"))
+
+ADC Interface Client - Entry Summary
```
[the remark this answers is not in the document]
Comment (iq-fred) [reply docs/manual/manual_adc_interface.rst:97]: Double check with Ben, but I think these can probably come out and we can just link to the pertinent API sections when they are done

**Review by iq-fred — APPROVED** [review]

**Pull request merged**